types/react-alert/index.d.ts

```diff
@@ -1,22 +1,23 @@
-// Type definitions for react-alert 5.2
+// Type definitions for react-alert 7.0
 // Project: https://github.com/schiehll/react-alert
 // Definitions by: Yue Yang <https://github.com/g1eny0ung>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-import * as React from 'react';
+// Note: The 7.x definition also applies to 6.x.
 
-export type AlertPosition =
+import { CSSProperties, ReactNode, ComponentType, Component, Context } from 'react';
+
+export type AlertPositionV4 =
     | 'top left'
     | 'top center'
     | 'top right'
-    | 'middle left'
-    | 'middle'
-    | 'middle right'
     | 'bottom left'
     | 'bottom center'
     | 'bottom right';
-
+export type AlertPosition = AlertPositionV4 | 'middle left' | 'middle' | 'middle right';
+export type AlertType = 'info' | 'success' | 'error';
+export type AlertTransition = 'fade' | 'scale';
 export interface Positions {
```

Comment:
just note, IMO those are actual symbols:
https://github.com/schiehll/react-alert/blob/master/src/options.js#L1
so in future we can rewrite this to `const`